lib/src/kbasesearchengine/main/NarrativeInfoDecorator.java

```diff
@@ -153,9 +188,17 @@ public GetObjectsOutput getObjects(final GetObjectsInput params, final String us
             }
         }
         catch (IOException | Auth2Exception e) {
-            LoggerFactory.getLogger(getClass()).error("ERROR: Failed retrieving workspace owner realname(s): " +
+            getLogger().error("ERROR: Failed retrieving workspace owner realname(s): " +
                             "setting to null: {}", e.getMessage());
         }
+
+        if(removedGuids != null && removedGuids.size() > 0){
```

Comment:
I think we always want to log removed guids, right? So removedGuids should never be null.